### ISSUE TITLE
Effects composition and CubeTexture background

### ISSUE THREAD

**Issue by bendxr** [vanruesc/postprocessing#105]
Hi, 
I'd like to combine a scene with several effects composition (outline + godRays + bloom) on the 3D meshes and a background (cubemap) which is not affected by all the effects.
The best I've achieved is to have both the scene and the background but rendered with all the effect passes. I tried several things to prevent effects to apply on the background but it didn't succeed until then.
Issue #79 seems to be quite close to my problem but it applied to processing 4.5 and I use 5.5.0 (TexturePass have disappeared from doc).
What should I try for solving this differentiated rendering issue ?

**Comment by vanruesc** [comment]
[WebGLBackground has been changed](https://github.com/mrdoob/three.js/pull/15662/commits/7bbd736d0d63efcb0f679263a2e2c0f9ea5b1c14) in `three@0.101.0`. The scene background no longer tests depth. This means you'll need to use a custom cube texture material to draw the background.

The `EffectPass` has also been changed in `postprocessing@5.6.0`. When there's no apparent need to render anything, it will now leave the internal frame buffers untouched.

The render pipeline changes as follows:

1. Render the 3D objects without the background.
2. Create a bloom effect and set its `blendFunction` to `SKIP`. Render the effect using a stand-alone `EffectPass`.
3. [Optional]: Render additional effects that should only be applied to the 3D objects.
   1. Use a [ShaderPass](https://vanruesc.github.io/postprocessing/public/docs/class/src/passes/ShaderPass.js~ShaderPass.html) together with a [CopyMaterial](https://vanruesc.github.io/postprocessing/public/docs/class/src/materials/CopyMaterial.js~CopyMaterial.html) to copy the input buffer over to the output buffer.
   2. Use a second `EffectPass` instance to render other effects.
4. Draw the background using a custom sky box with a material that tests depth.
5. Draw the bloom as a `TextureEffect`.

[I've updated the example accordingly.](https://jsfiddle.net/v9yurh3x/1/)
